frontend/src/components/Status.vue

```diff
@@ -1,52 +1,64 @@
 <template>
-  <div
-    v-if="!isLogin && user"
-    class="is-online"
-    :class="{ online: isOnline }"
-  />
+  <div v-if="!isLogin && user">
+    <div v-if="isOnline" class="online" @click="">
```

Comment:
```suggestion
    <div v-if="isOnline" class="online">
```
or am I missing something?